docs/docs/developer-guides/cosmwasm-cli.md

```diff
@@ -40,6 +58,20 @@ This will take time at first but since it is cached, it will be almost instant a
 ```
 nix run "github:ComposableFi/composable/d2845fc731bc3ee418a17cf528336d50f4b39924#devnet-picasso"
```

Comment:
do not refer commits, either refer release tags or branches.

Reply:
commits just for internal use and ugly looking

Reply:
it is just an example